pkg/reconcile/pipeline/infinispan/handler/manage/upgrades.go

```diff
@@ -141,18 +142,80 @@ func GracefulShutdown(i *ispnv1.Infinispan, ctx pipeline.Context) {
 					return
 				}
 
-				var rebalanceDisabled bool
+				type PodMeta struct {
```

Comment:
Should we name it `ServerMeta`? I think `PodMeta` is often used in relation to k8s Pod Metadata and I'd expect the information from those in here. May confuse us in the future